src/main.ts

```diff
@@ -1,11 +1,20 @@
-import { enableProdMode } from "@angular/core";
-import { platformBrowserDynamic } from "@angular/platform-browser-dynamic";
+import { enableProdMode } from '@angular/core';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 
-import { AppModule } from "./app/app.module";
-import { environment } from "./environments/environment";
+import { AppModule } from './app/app.module';
+import { environment } from './environments/environment';
 
 if (environment.production) {
 	enableProdMode();
 }
 
+// Uncomment the following lines to activate the service worker
```

Comment:
Can we remove this? Do we need it?

Reply:
Yes we do. We don't want the service worker to be active by default because the caching makes development not as smooth and we want people to be very easily able to activate them by uncommenting those lines.